tests/scripts/install.sh

```diff
@@ -61,7 +61,7 @@ if [ -z "${OPENSHIFT_USER}" ] || [ -z "${OPENSHIFT_PASS}" ]; then
 fi
 
 echo "Creating the following KfDef"
-cat ./kfctl_openshift.yaml
+cat ./kfctl_openshift.yaml | tee ${ARTIFACT_DIR}/kfctl_openshift.yaml
```

Comment:
Same here, does `cp` make more sense?

Reply:
After a chat with @crobby, I have removed the output of logs/files/... that we are saving as artifacts